autopts/ptsprojects/stack/layers/bap.py

```diff
@@ -26,7 +26,9 @@ def __init__(self):
         self.peers = {}
         self.ase_configs = []
         self.broadcast_id = 0x1000000  # Invalid Broadcast ID
+        self.broadcast_id_2 = 0x1000000  # Invalid Broadcast ID 2
         self.broadcast_code = ''
+        self.wid_counter = 0
```

Comment:
I would rename this to 
```suggestion
        self.hdl_wid_114 = 0
```

It is specifically used for this WID and if multiple WID handles uses this with the generic name, then we may end up with an unexpected count